src/utils.cpp

```diff
@@ -693,6 +693,30 @@ std::vector<std::pair<std::string, std::string>> get_cgroup_paths(
   return result;
 }
 
+bool is_loaded_module(const std::string &module)
+{
+  if (module == "vmlinux")
+  {
+    return true;
+  }
+
+  // This file lists all loaded modules
+  std::ifstream modules_file("/proc/modules");
+
+  for (std::string line; std::getline(modules_file, line);)
+  {
+    std::string first_token = line.substr(0, line.find(' '));
+    if (module == first_token)
```

Comment:
Nit: this should be slightly faster as it requires just one pass through the string:
```suggestion
    if (line.compare(0, module.size() + 1, module + " ") == 0)
```